tests/foreman/ui/test_role.py

```diff
@@ -323,78 +335,88 @@ def test_positive_create_overridable_filter(self):
                 menu_locators['menu.hosts'], timeout=3))
 
     @stubbed
-    @tier1
+    @tier2
     def test_positive_update_role_taxonomies(self):
-        """Update role taxonomies which applies to its non-overrided filters
+        """Update role taxonomies which applies only to non-overrided filters
+        in role
 
         @id: f705faea-0d1c-4c11-b82d-b6b0e848f75d
 
         @steps:
 
-        1. Update existing role with different taxonomies
-
-        @assert: The taxonomies are applied only to non-overrided role filters
-
-        @caseautomation: notautomated
-        """
-
-    @stubbed
-    @tier1
-    def test_negative_update_role_taxonomies(self):
-        """Update role taxonomies which doesnt applies to its overrided filters
-
-        @id: 9ffca18f-8784-48e2-8972-a67979a44508
+        1. Create role with overrided, non-overrided resources and assign some
+            taxonomies (organizations and locations) to role
+        2. Assign this role to user
+        2. Update existing role with different taxonomies
```

Comment:
3. here, and the other one found by alex. @jyejare feel free to merge after fixing